### PR TITLE
[13.0][MIG] operating unit

### PR DESCRIPTION
**Pull request by alan196**

### CODE REVIEW

File: operating_unit/models/operating_unit.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    active = fields.Boolean(default=True)
+    company_id = fields.Many2one(
+        "res.company",
+        required=True,
+        readonly=True,
+        default=lambda self: self.env.company,
+    )
+    partner_id = fields.Many2one("res.partner", "Partner", required=True)
+    user_ids = fields.Many2many(
+        "res.users", "operating_unit_users_rel", "poid", "user_id", "Users Allowed"
```
Comment (AaronHForgeFlow) [inline operating_unit/models/operating_unit.py]: I think version 13 is a good moment to put a good column name here. It should be something like `operating_unit_id` or `ou_id`. Migration script could be added later, after the merge.

**Review by AaronHForgeFlow — COMMENTED** [review]
@alan196 Thanks for migrating this. It seems to work fine. Could you split the migration commit in two? First commit with the isort,black and all the syntax stuff. And the second one with the actual migration. That is easy for reviewing.

Thanks.

File: operating_unit/models/operating_unit.py (hunk 1 above)
Comment (alan196) [reply operating_unit/models/operating_unit.py]: Thanks for your review! I already rename the column to operating_unit_id

**Comment by alan196** [comment]
@AaronHForgeFlow I already split the commit in two. 👍

File: operating_unit/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+# Copyright 2015-TODAY Serpent Consulting Services Pvt. Ltd. - Sudhir Arya
+# License: LGPL-3 or later (https://www.gnu.org/licenses/lgpl.html).
+{
+    "name": "Operating Unit",
+    "summary": "An operating unit (OU) is an organizational entity part of a "
+    "company",
+    "version": "13.0.1.0.0",
+    "author": "Eficent, "
+    "Serpent Consulting Services Pvt. Ltd.,"
+    "Jarsa Sistemas, "
```
Comment (AaronHForgeFlow) [inline operating_unit/__manifest__.py]: For migrating modules without adding any major feature putting yourself in the contributors section in the readme should be enough. This is reserved for the original author + contributors who do major functional changes.

Hope you understand.

Comment (alan196) [reply operating_unit/__manifest__.py]: Thanks for the clarification 👍
I already make the change

Comment (AaronHForgeFlow) [reply operating_unit/__manifest__.py]: Thank you!

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Tested + code. Thanks.

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM :+1:

**Review by lmgr0312 — APPROVED** [review]
LGTM.. :+1:

**Comment by BT-jortells** [comment]
@alan196 @AaronHForgeFlow @LoisRForgeFlow 

The issues specified in [V12 Known issues / Roadmap](https://github.com/OCA/operating-unit/blob/12.0/operating_unit/README.rst#known-issues--roadmap) have been solved only partially in this PR, since the migration script has not been developed. So, an update from v12 to v13 will result in an error because of the table column name changed

**Comment by AaronHForgeFlow** [comment]
@BT-jortells Yes, you are right. It is still pending. I don't have time right now for this, as I am not migrating to v13 now. But, eventually, it has to be done.

**Comment by BT-jortells** [comment]
Thanks for answering. I have also referenced this in the [Migration to version 13.0 issue](https://github.com/OCA/operating-unit/issues/200) to keep it in mind.

**Pull request merged**